### PR TITLE
Added summary report quickstart guide

### PR DESCRIPTION
**Pull request by maudnals**
https://deploy-preview-4246--developer-chrome-com.netlify.app/docs/privacy-sandbox/summary-reports/design-decisions

### CODE REVIEW

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 1]
```diff
@@ -0,0 +1,766 @@
...
+updated: 2022-11-09
+description: >
+  Use Noise Lab to preview how noise affects your summary reports.
+tags:
+  - privacy
+---
+
+## About this document
+
+Readers will:
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:22]:
```suggestion
When you read this article, you will:
```

Comment (heyawhite) [reply site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:22]: A little more informal and welcoming.

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 2]
```diff
@@ -0,0 +1,766 @@
...
+**Add**/**Remove**/**Reset** buttons.
+
+{% endDetails %}
+{% endAside %}
+
+If you measure one data point (measurement goal) on a conversion event, such as
+conversion count, that data point can get all the contribution budget (65536). If you set multiple measurement goals on a conversion event,
+such as conversion count and purchase value, then those data points will need to
+share the contribution budget. This means you have less leeway to scale up your
+values. **Therefore, the more measurement goals you have, the lower signal-to-noise ratios are likely to be (higher noise)**.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:415]:
```suggestion
values. 

**Therefore, the more measurement goals you have, the lower signal-to-noise ratios
are likely to be (higher noise)**.
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 3]
```diff
@@ -0,0 +1,766 @@
...
+
+Another decision to make regarding measurement goals is the budget split. If you split the contribution budget equally across two data points, each data point gets a
+budget of 65536/2 = 32768. This may or may not be optimal depending on the
+maximum possible value for each data point. For example, if you're measuring
+purchase count that has a maximum value of 1, and purchase value with a
+minimum of 1 and a maximum of 120, the purchase value would benefit from having
+"more space" to be scaled up—that is, to be given a larger proportion of the
+contribution budget. **See if some measurement goals need to be prioritized over
+others in relation to the impact of noise.**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:424]:
```suggestion
contribution budget. **You'll see if some measurement goals should be prioritized over
others in relation to the impact of noise.**
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 4]
```diff
@@ -0,0 +1,766 @@
...
+
+{% endDetailsSummary %}
+
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+A measurement goal is a distinct data point collected in conversion events.
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:442]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 5]
```diff
@@ -0,0 +1,766 @@
...
+    if you want to see an expanded version of the table (#_5. in the screenshot below)_
+  - Below each data table in the output section, there is an option
+    to download the table for offline usage. Additionally, in the bottom
+    right corner there is an option to download all data tables (#_6. in
+    the screenshot below_)
+- Test different settings for the parameters in the **Parameters** section
+  and click **Simulate** to see how they impact the output:
+
+<figure>
+{% Img src="image/EvKqfXr4ercavLaApW41d8uGXN72/8o20EoPwoN2fxoNHPg82.jpg", alt="Screenshot: Noise Lab interface for Simple mode", width="800", height="450" %}
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:542]:
```suggestion
{% Img src="image/EvKqfXr4ercavLaApW41d8uGXN72/8o20EoPwoN2fxoNHPg82.jpg", alt="Noise Lab interface for Simple mode", width="800", height="450" %}
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 6]
```diff
@@ -0,0 +1,766 @@
...
+  - To test out different Key Structures, switch **Key Strategy** to "B"
+  - Input the number of different key structures you want to use
+    (default is set to "2")
+  - Click **Generate Key Structures**
+  - You will see options to specify your key structures by clicking
+    the checkboxes next to the keys you want to include for each key structure
+  - Click **Simulate** to see the output.
+
+<figure>
+{% Img src="image/EvKqfXr4ercavLaApW41d8uGXN72/5qH8KUvKhhW4vbUbNzbT.jpg", alt="Screenshot: Noise Lab interface for Advanced mode", width="800", height="450" %}
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:565]:
```suggestion
{% Img
  src="image/EvKqfXr4ercavLaApW41d8uGXN72/5qH8KUvKhhW4vbUbNzbT.jpg",
  alt="Advanced mode offers controls for measurement goals and dimensions to track, highlighted in the sidebar.",
  width="800", height="450"
%}
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 7]
```diff
@@ -0,0 +1,766 @@
...
+  <figcaption>
+    Noise Lab interface for Advanced mode.
+  </figcaption>
+</figure>
+
+<figure>
+{% Img src="image/EvKqfXr4ercavLaApW41d8uGXN72/DfUCCLzvay9SgExqUMc0.jpg", alt="Screenshot: Noise Lab interface for Advanced mode", width="800", height="450" %}
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:572]:
```suggestion
{% Img
  src="image/EvKqfXr4ercavLaApW41d8uGXN72/DfUCCLzvay9SgExqUMc0.jpg",
  alt="Advanced mode also a Key strategy option in the Parameters section of the sidebar.",
  width="800", height="450"
%}
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md (hunk 4 above)
Comment (heyawhite) [reply site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:442]: The section heading is "Try it out", so no need for repeat!

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 8]
```diff
@@ -0,0 +1,766 @@
...
+
+{% endDetailsSummary %}
+
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+A measurement goal is a distinct data point collected in conversion events.
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:365]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 9]
```diff
@@ -0,0 +1,766 @@
...
+
+#### Try it out in Noise Lab 🧪
+
+{% endDetailsSummary %}
+
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:320]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 10]
```diff
@@ -0,0 +1,766 @@
...
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+While this can be difficult to predict and can have significant variations in
+addition to seasonality effects, try and estimate the number of daily
+single-touch attributable conversions to the closest power of 10: 10, 100,
+1,000, or 10,000.
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:261]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 11]
```diff
@@ -0,0 +1,766 @@
...
+#### Try it out in Noise Lab 🧪
+
+{% endDetailsSummary %}
+
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+**Try it out:**
+
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:212]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 12]
```diff
@@ -0,0 +1,766 @@
...
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+In the **Simple** mode, the default key structure is used. In the **Advanced**
+mode, you can experiment with different key structures. Some example dimensions
+are included; you can also modify these.
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:147]:
```suggestion
```

File: site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md [hunk 13]
```diff
@@ -0,0 +1,766 @@
...
+
+#### Try it out in Noise Lab 🧪
+
+{% endDetailsSummary %}
+
+{% Aside %}
+New to [Noise Lab](https://noise-lab.uc.r.appspot.com/)? [Take a quick tour](#quick-tour).
+{% endAside %}
+
+**Try it out:**
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:81]:
```suggestion
```

Comment (heyawhite) [reply site/en/docs/privacy-sandbox/summary-reports/design-decisions/index.md:81]: Removing this repetition across the doc.

**Review by heyawhite — APPROVED** [review]
Tiny nits which I'll commit and merge!

**Comment by maudnals** [comment]
Thank you!!

**Pull request merged**